app/controllers/users_controller.rb

```diff
@@ -36,9 +85,17 @@ def load_user
     redirect_to root_path
   end
 
-  private
   def user_params
     params.require(:user).permit :name, :email, :password,
                                  :password_confirmation
   end
+
+  def admin_user
+    action_if_not_admin unless current_user.admin?
+  end
+
+  def action_if_not_admin
+    redirect_to root_url
+    flash[:danger] = t "notification.not_admin"
+  end
```

Comment:
```suggestion
  def admin_user
    return if current_user.admin?
    
    redirect_to root_url
    flash[:danger] = t "notification.not_admin"
  end
```
mắc chi phải tạo ra 1 method mới vậy cậu, có dùng lại `action_if_not_admin` này đâu